tools/woff/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "vexflow-fonts",
+  "version": "1.0.1",
+  "description": "Web fonts compatible with VexFlow.",
+  "author": "",
+  "license": "SEE LICENSE.txt in each font folder."
+}
```

Comment:
@AaronDavidNewman @0xfe 
Since we are redistributing open source fonts on the unpkg CDN, I figured it'd be good practice to include the LICENSE files from each respective project.

https://unpkg.com/vexflow-fonts@1.0.1/

These can be the web font files that VexFlow supports by default. A developer can choose his/her own (e.g., from Google Fonts) and then host on a different CDN.

I'm keeping these font files in the vexflow/tools/woff/ folder, but publishing them as a separate npm project. This allows us to update font files (if necessary) without having to push a new vexflow release.